underactuated/exercises/fully_actuated_vs_underactuated/trajectory_tracking_state_space.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
will merge this now, but for every .py / .ipynb file, please add an entry in a `BUILD.bazel` file.  This will add it as a unit test (to make sure that it works, and continues to work).  i'll ask you to include that in a follow-up PR?

Reply:
I see. Ok, no problem!